index.html

```diff
@@ -109,7 +109,7 @@ <h1>Dynamic Elasticsearch Profile Flame Graph</h1>
             const colorScale = d3.scaleOrdinal(d3.schemeCategory10);
 
             const partition = d3.partition()
-                .size([width, height])
+                .size([height, width])
```

Comment:
original documentation states `[width, height]`  is correct, how is your version fixed the problem you described?
https://d3js.org/d3-hierarchy/partition#partition_size

Reply:
This usually happens when coordinate plane has swapped x and y coordinates. It could be either a bug in the code, or intentional rotation operation. I didn't dig the root issue.